hpc0.qmd

```diff
@@ -1,297 +1,1300 @@
-- Hands on experience and confidence building
-- Basic interactions with files and directories: command line execution, directory and file basics, files properties.
-- File manipulation: inspecting and editing files, searching and filtering,
-- Scripting and workflows: scripting techniques for building workflows.
 
-# Introductions and Motivations
+## Logging in 1/3
+- [ARC Documentation >> Login]({{< var rc.login >}})
+- How you login depends on your:
+  - Host Operating System
+  - Connection
 
-Who are you and why are you here?
+::: {.notes}
+- "The Website provides details about logging in"
+- "How you login depends on your host Operating System and connection..."
+:::
 
-## Me {visibility="hidden"}
 
-- Research Computing service
-- Research Computing Consultant
-- Everything from HPC to Data Science
-- RSE Cloud Computing Fellow
-- Research in Deep Learning: Neural networks for text summarisation
+## Logging in 2/3
+- Host Operating System:
+  - OSx or Linux:
+    - Use `terminal`
+  - Windows:
+    - Use `MobaXTerm`
+
+::: {.notes}
+- "The OSx `terminal` is already a Linux command line interface program:"
+  - "Which may work slightly differently from the HPC Linux command line, but it will be broadly similar."
+- "`MobaXTerm` is software that you should be able to download, install and run without needing help from systems administators."
+  - "It provides a sort of bridge between your host Windows machine and other machines you connect with."
+- Maybe conduct a poll:
+  - "What host OS are you using?"
+:::
 
-# Linux Basics {visibility="hidden"}
 
-## Logging in... {background-color="#202020"}
+## Logging in 3/3
+- Connection
+  - On campus
+    - Wireless is like 'off campus'
+  - Off campus
+    - But connected to the [University VPN](https://it.leeds.ac.uk/it?id=kb_article&sysparm_article=KB0014410) is like 'on campus'
 
-Using MobaXTerm
+::: {.notes}
+- Read slide:
+  - "In terms of connection: On campus, but using wifi is like being 'off campus'..." 
+- The University Virtual Private Network (VPN) is a type of secure network connection, enabling your computer to access the University network when working remotely.
+- Maybe conduct a poll:
+  - "What type of connection do you have?"
+:::
+
+
+## Logging in demos
+
+::: {.notes}
+- Demo login from OSx/Linux.
+- Demo login from Windows.
+:::
 
-## What is BASH?
 
+## Logging in exercise
+- Please login.
+
+::: {.notes}
+- Maybe conduct a poll:
+  - "Have you logged in?"
+- Conduct/organise help if necessary...
+  - Maybe susggest one-to-ones if there are helpers...
+- Those that cannot login can still follow along for a bit...
+- Is it time for a break 1?
+:::
+
+
+## The shell interface {.smaller}
+- You login to a ('BASH') 'shell' and your 'Home Directory'
+- A 'shell' is a terminal emulator program.
+  - It is a window with a `prompt` at which textual commands are entered.
+  - Standard Output (std.out) and Standard Error (std.err) streams get displayed in the window.
```

Comment:
std.out and std.err don't strike me as normal terms.  stdout, stderr likely better here?

Reply:
std.out and std.err are still evil.